recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conans import ConanFile, CMake
+from conan.tools.files import get
+
+
+class OctoLoggerCPPConan(ConanFile):
+    name = "octo-logger-cpp"
+    license = "MIT"
+    url = "https://github.com/ofiriluz/octo-logger-cpp"
+    homepage = "https://github.com/ofiriluz/octo-logger-cpp"
+    description = "Octo logger library"
+    topics = ("logging", "cpp")
+    generators = "cmake", "cmake_find_package_multi"
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
+
+    def source(self):
+        get(self, **self.conan_data["sources"][str(self.version)], strip_root=True, destination=self._source_subfolder)
+
+    def requirements(self):
+        self.requires("catch2/3.1.0")
+        self.requires("fmt/9.0.0")
+        self.requires("trompeloeil/42")
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder, build_folder=self._build_subfolder)
+        cmake.build(build_dir=self._build_subfolder)
+        cmake.test(build_dir=self._build_subfolder)
```

Comment:
```suggestion
```
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-conancenter-does-not-build-and-execute-tests-in-recipes

Reply:
fixed